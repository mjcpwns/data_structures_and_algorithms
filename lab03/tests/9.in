1 10 4 2 67 8 3 5 6