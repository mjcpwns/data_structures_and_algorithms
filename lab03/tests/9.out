Rotating left
Rotating left
Rotating left
Rotating left
Rotating right
Tree:
      4
     / \
    /   \
   /     \
  2      10
 / \     / \
1   3   6  67
       / \
      5   8

#nodes = 9,  #leaves = 5
Infix  : 1 2 3 4 5 6 8 10 67 
Prefix : 4 2 1 3 10 6 5 8 67 
Postfix: 1 3 2 5 8 6 67 10 4 
ByLevel: 4 2 10 1 3 6 67 5 8 
